src/components/ActionProfilesForm/ActionProfilesForm.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Headline,
+  TextArea,
+  TextField,
+  Accordion,
+  AccordionSet,
+} from '@folio/stripes/components';
+import stripesForm from '@folio/stripes/form';
+
+import { FullScreenForm } from '../FullScreenForm';
+import { validateRequiredField } from '../../utils';
+
+const formName = 'actionProfilesForm';
+
+export const ActionProfilesFormComponent = props => {
+  const {
+    pristine,
+    submitting,
+    handleSubmit,
+    onCancel,
+  } = props;
+
+  const isSubmitDisabled = pristine || submitting;
+
+  const paneTitle = <FormattedMessage id="ui-data-import.settings.actionProfiles.new" />;
+  const headLine = <FormattedMessage id="ui-data-import.settings.actionProfiles.new" />;
```

Comment:
I would just set it to equal `paneTitle` as it is the same.

Reply:
Done